reservations/models.py

```diff
@@ -432,6 +432,40 @@ def requires_handling(self) -> bool:
             self.reservation_unit.filter(require_reservation_handling=True).exists() or self.applying_for_free_of_charge
         )
 
+    @property
+    def reservee_name(self) -> str:
+        # Internal recurring reservations are created by STAFF
+        if self.type == ReservationTypeChoice.STAFF.value and self.recurring_reservation is not None:
+            return self.recurring_reservation.name
+
+        # Blocking reservation
+        if self.type == ReservationTypeChoice.BLOCKED.value:
+            return _("Closed")
```

Comment:
Also need the locale message strings:
- fi: "Suljettu"
- sv: "Stängt"

Reply:
Added translations for this, but something to consider is that we are currently missing ~99% of translations. :)